### DISCUSSION TITLE
Publish public Mattermost channels to the Internet

### DISCUSSION THREAD

**Discussion by chenilim** [mattermost/community#3] [Practical Ideas]
This is one of those things where "this must already exist, right?" But in case it doesn't, there should be a plugin or button that publishes public channels on the Internet, so that it can be SEO-craweled and accessible without a login.

Ideally, the channels should be organized by thread, and maybe displayed like discussion threads (like here in GitHub).

**Comment by jprusch** [comment]
Hehe, about the same idea... https://github.com/mattermost/community/discussions/24

**Reply by BenLloydPearson** [reply]
I also mentioned this in the other thread, so I will here too. I think this is a great idea, we could use this to display a live feed at developer events that shows the activities we have happening on our chat server during the event.

**Comment by bbodenmiller** [comment]
Would be great if even just a channel directory could be published to a webpage as a starting point (sort of related to https://github.com/mattermost/community/discussions/118).

**Comment by esethna** [comment]
Yay for default (even-more) open!
